### ISSUE TITLE
Unexpected/unlisted hotkey

### ISSUE THREAD

**Issue by YI8it** [digimezzo/dopamine-windows#1366]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report  
[ ] Feature request
</code></pre>

## I'm using these versions

* Windows version: 10
* Dopamine version: 2.0.9

## Reproduction steps

* Step 1 Press back/forward on mouse
* Step 2 Read Help
## Observed behavior

Dopamine changes song on mouse back/forward
## Desired behavior

As this is not a hotkey listed under help, this should not happen. As it is now I would expect it to do nothing, and that is why I listed it as a bug. What behavior I desire is irrelevant as configuring hotkeys is a waste of developer time, but all available hotkeys should be listed (what I mean by waste of time is that it's impossible to cover user needs when it comes to hotkeys, so letting users set them is the way to go, in my opinion).
<!--

**Comment by digimezzo** [comment]
Unfortunately, it is not a trivial as it sounds. I never explicitly programmed it to react on those mouse clicks. This seems to be an automatic side effect of the back and forward arrow keys support. .NET probably has these mapped to the same event.

**Comment by th3v4n** [comment]
Yeah I would prefer if they worked back and forward. It'd usually be Alt+Left/Right arrow.
